devtools/create_project/xcode.cpp

```diff
@@ -69,6 +69,31 @@ namespace CreateProjectTool {
 	_buildFile._flags = kSettingsSingleItem; \
 }
 
+/**
+ * Checks whether a file or directory exists.
+ */
+bool exists(const char * const fileName) {
+	struct stat unused;
+	return (stat(fileName, &unused) == 0);
+}
+
+const char * const DIRECTORIES[] = {
+	/* Homebrew */ "/usr/local/",
+	/* MacPorts */ "/opt/local/",
+	/* Fink     */ "/sw/"
+};
+
+std::string findPath(const char * const lib) {
+	for (int i = 0; i < 3; ++i) {
```

Comment:
While I don’t like the code duplication, I will just add it to this file since it is a one-liner.